chain/consensus/mir/mine.go

```diff
@@ -10,20 +10,24 @@ import (
 	"golang.org/x/xerrors"
 
 	"github.com/filecoin-project/go-address"
-	"github.com/filecoin-project/mir"
-	mirproto "github.com/filecoin-project/mir/pkg/pb/requestpb"
-
 	"github.com/filecoin-project/lotus/api/v1api"
 	"github.com/filecoin-project/lotus/chain/consensus/mir/db"
+	"github.com/filecoin-project/lotus/chain/consensus/mir/validator"
+	"github.com/filecoin-project/mir"
+	mirproto "github.com/filecoin-project/mir/pkg/pb/requestpb"
 )
 
 const (
 	ReconfigurationInterval = 2000 * time.Millisecond
 )
 
-var (
-	ErrMirCtxCanceledWhileWaitingSnapshot = fmt.Errorf("context canceled while wating for a snapshot")
-)
+type ErrMirCtxCanceledWhileWaitingSnapshot struct {
```

Comment:
We use this error in places other than the snapshot, right? 
```suggestion
type ErrMirCtxCanceledWhileWaitingBlock struct {
```

Reply:
correct.